packages/plugins/src/plugins/eigenlayer/fetchers/withdrawal.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  NetworkId,
+  PortfolioElement,
+  PortfolioElementType,
+} from '@sonarwatch/portfolio-core';
+import { Cache } from '../../../Cache';
+import { Position, Withdrawal } from '../types';
+import { platformId } from '../constants';
+import { getAddress } from 'viem';
+
+import { getAssetsFromPositions } from '../helper';
+
+/**
+ * Returns the withdrawals for a given owner
+ * @param owner - The address of the owner
+ * @param cache - The cache instance
+ * @returns The withdrawals
+ */
+export const getWithdrawals = async (owner: string, cache: Cache) => {
+  const withdrawals = await cache.getItem<Withdrawal[]>(
+    'eigenlayer-withdrawals',
+    {
+      prefix: platformId,
+      networkId: NetworkId.ethereum,
+    }
+  );
+
+  const strategies = await cache.getItem<Position[]>('eigenlayer-strategies', {
+    prefix: platformId,
+    networkId: NetworkId.ethereum,
+  });
+
+  const userWithdrawals = withdrawals?.filter(
+    (withdrawal) =>
+      getAddress(withdrawal.withdrawerAddress) === getAddress(owner)
```

Comment:
If `owner` is already of `Address` type, could we simplify the below to :

```suggestion
      withdrawal.withdrawerAddress ===owner
```
?

Reply:
`getAddress` is also checksumming both addresses, the `Address` type only check if the string start by `0x`. I need to add it to be sure that both address I compare are checksummed and the same